server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/SQLQueryBuilder.java

```diff
@@ -134,6 +132,25 @@ public String buildInsertQuery() throws SQLException {
         return sb.toString();
     }
 
+    /**
+     * Build query to set environment variables in external database.

```

Comment:
is this statement more accurate?
```suggestion
     * Build query to set session-level variables in external database.
```